src/playlist-selectors.js

```diff
@@ -284,8 +284,35 @@ export let simpleSelector = function(
     resolutionPlusOneRep = resolutionPlusOneSmallest.filter((rep) => rep.bandwidth === highestRemainingBandwidthRep.bandwidth)[0];
   }
 
+  let leastPixelDiffRep;
+
+  // If this selector is proven to be better than others.
+  // resolutionPlusOneRep and resoltionBestRep and all
+  // the code involving them should be removed.
+  if (masterPlaylistController.experimentalPixelDiffSelector) {
+    // find the smallest variant that is larger than the player
+    // if there is no match of exact resolution
```

Comment:
This comment is exactly the same as the one for `resolutionPlusOneList`, but the behavior is different. Maybe each should call out why they're choosing each method?